compiler-plugin-test/src/test/resources/codeaction/mandatory-relation-entity-type-1.bal

```diff
@@ -0,0 +1,77 @@
+import ballerina/persist as _;
+
+type Building record {|
+    readonly string buildingCode;
+    string city;
+    string state;
+    string country;
+    string postalCode;
+    Workspace[] workspaces;
+|};
+
+type Workspace record {|
+    readonly string workspaceId;
+    string workspaceType;
+|};
+
+type Building1 record {|
+    readonly string buildingCode;
+    string city;
+    string state;
+    string country;
+    string postalCode;
+	Workspace1? workspace1;
```

Comment:
```suggestion
    string postalCode;
    Workspace1? workspace1;
```

Reply:
This seems to be a bug in Github PR preview. Locally this is generated correctly.